### ISSUE TITLE
No such cmd XX on object X

### ISSUE THREAD

**Issue by law-abiding-citizen** [chrippa/xmmsclient.js#1]
server.js
var xmmsclient = require('xmmsclient');
var player = new xmmsclient.Client('myclient');
player.connect('tcp://127.0.0.1:7890');
player.playlist.add_url(name, url);

Upon node server.js (Windows) , on console (on which xmms2d.exe -i tcp://127.0.0.1:7890 is working):
Client connected
No such cmd 35 on object 2
No such cmd 32 on object 1
disconnect was true!
Destroying client!

**Comment by chrippa** [comment]
Hmm, which version of XMMS2 do you use? The library only work with latest git head of xmms2 currently.

**Comment by law-abiding-citizen** [comment]
I am using the only win32 binaries available. I guess they are way too old.

![xmmsversion](https://cloud.githubusercontent.com/assets/2619563/2873873/e26037d6-d3c3-11e3-8d2b-729a3cfafece.png)

**Comment by chrippa** [comment]
Yeah, that looks ancient. I don't think there are any newer binaries available though. :-/
You might be able to build a new version with mingw but I have not tried myself.
